iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -141,6 +65,73 @@ public struct AddMealView: View {
             }
         }
     }
+
+    func createTextFields() -> some View {
```

Comment:
I think Apple recommend just creating a new struct that conforms to View for stuff like this since it's not expensive to do, I just don't know how complicated that makes it when using view models